src/bussinessLogic/ManageMap.java

```diff
@@ -0,0 +1,100 @@
+package bussinessLogic;
+
+import interfaces.MapManager;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class ManageMap implements MapManager {
+    // the number of letters in the alphabet
+    private final int numberOfLetters = 3;
+    // empty hash map for storing the alphabet and adding the reaction time for each
+    // character
+    LinkedHashMap<String, Integer> charMap = new LinkedHashMap<>();
+    // hash map for storing the number of times a char is displayed (needed for
+    // calculating the average)
+    LinkedHashMap<String, Integer> dividerMap = new LinkedHashMap<>();
+    LinkedHashMap<String, Integer> sortedMap;
+    // current char
+    private char currentChar;
+    // create a string to store the chars when needed as string
+    private String charString;
+
+    /**
+     * creates a new charMap and ads the chars to the map
+     */
+    public void setNewCharMap() {
+        // Add elements to the map
+        currentChar = 'a';
+
+        for (int i = 0; i < numberOfLetters; i++) {
+            charString = Character.toString(currentChar);
+            charMap.put(charString, 0);
+            dividerMap.put(charString, 0);
+            currentChar++;
+        }
+    }
+
+    /**
+     * sort the charMap in ascending order
+     */
+    public void sortCharMap() {
```

Comment:
The method can be made private and called from the constructor of this class.

You can also take into consideration creating the _CharMap_ entity already sorted.

Reply:
I can't do that. `CharMap` depends on the speed each key is typed and only after that the map is sorted. I shall rename the variables because some of them are not very explicit or even misleading

Reply:
Can the first part of the affirmation be achieved?

> The method can be made private and called from the constructor of this class.